### ISSUE TITLE
ImportError: Unfortunately aarch64 is not currently supported

### ISSUE THREAD

**Issue by nilnullzip** [dlshriver/intercepts#7]
Love your package! Ran into this error on ARM and I'm trying to understand how to fix. I rand handler.c and got string `b201ebe3b201f3e0d61f0060`. I'm stuck on this code in linux.py:

```
INSTR_TEMPLATE = bytes.fromhex("48bfaaaaaaaaaaaaaaaa48b8bbbbbbbbbbbbbbbbffe0")
_i = INSTR_TEMPLATE.index(b"\xbb" * 8)
INSTR_TEMPLATE = INSTR_TEMPLATE[:_i] + PyObject_Call_address + INSTR_TEMPLATE[_i + 8 :]
```

I gather I have to do something similar to substitute `PyObject_Call_address` into the ARM code. Is this all that is needed? I have not yet found a way to disassemble the ARM hex code. Any pointers there? Any other suggestions?

**Comment by dlshriver** [comment]
Unfortunately, it is not nearly as straightforward to perform this substitution with ARM code. The width of instructions means that constants (like `0xaaaaaaaaaaaaaaaa`) are generally broken up into smaller pieces, or the compiler uses tricks to optimize the loading of these values into the registers that make them difficult to simply replace in generated code. If you want to add this feature, my main suggestion is to become extremely familiar with the ARM instruction set. Hopefully there's some simple trick to implementing this, but I'm not sure at the moment.

**Comment by nilnullzip** [comment]
For the record, I found this disassembler useful in implementing pull request #8:

> https://shell-storm.org/online/Online-Assembler-and-Disassembler/?opcodes=c30000586000005860001fd61f2003d5aaaaaaaaaaaaaaaabbbbbbbbbbbbbbbbcccccccccccccccc&arch=arm64&endianness=little&baddr=0x00000000&dis_with_addr=True&dis_with_raw=True&dis_with_ins=True#disassembly

**Comment by dlshriver** [comment]
Resolved by #8. Thank you for your great contribution!
